src/pkg/packager/deploy.go

```diff
@@ -38,9 +38,16 @@ var (
 )
 
 // Deploy attempts to deploy the given PackageConfig.
-func (p *Packager) Deploy() error {
+func (p *Packager) Deploy() (err error) {
 	message.Debug("packager.Deploy()")
 
+	// Attempt to connect to a Kubernetes cluster.
+	// Not all packages require Kubernetes, so we only want to log a debug message rather than return the error when we can't connect to a cluster.
+	p.cluster, err = cluster.NewCluster()
+	if err != nil {
+		message.Debug(err)
+	}
+
```

Comment:
Since we are doing this now we should also refactor line 60 of `src/pkg/packager/interactive.go` to use p.cluster and simply check if it is nil instead of instantiating it's own cluster.